Python/pywarpx/picmi.py

```diff
@@ -137,6 +137,10 @@ class Species(picmistandard.PICMI_Species):
         Cells with fewer particles than this number will be
         skipped during resampling.
 
+    warpx_resampling_algorithm_max_w: float
```

Comment:
For the sake of clarity I think it would be better to name this `warpx_resampling_algorithm_max_weight`.